cmd/flowserver/flow.go

```diff
@@ -44,10 +44,11 @@ func (r *flowResponse) MarshalJSON() ([]byte, error) {
 }
 
 type startRequest struct {
-	Flows   json.RawMessage      `json:"flows"    validate:"required"`
-	Contact json.RawMessage      `json:"contact"  validate:"required"`
-	Extra   json.RawMessage      `json:"extra,omitempty"`
-	Input   *utils.TypedEnvelope `json:"input"`
+	Environment *json.RawMessage     `json:"env"`
```

Comment:
More of a discussion point but do we want to go down the line of abbreviating some things or keep everything verbose? My bias would be keep things verbose apart from acronyms but what do you guys think? @ericnewcomer